quickwit/quickwit-serve/src/otlp_api/rest_handler.rs

```diff
@@ -47,6 +48,7 @@ pub(crate) fn otlp_ingest_api_handlers(
         .or(otlp_default_traces_handler(otlp_traces_service.clone()))
         .or(otlp_logs_handler(otlp_logs_service))
         .or(otlp_ingest_traces_handler(otlp_traces_service))
+        .recover(recover_fn)
```

Comment:
the otlp routes could be handled individually, they don't look to me like they overlap

Reply:
agreed. I moved it inside.